### PR TITLE
Fix debian package builds.

### PR DESCRIPTION
**Pull request by erikjohnston**
This was due to dh-virtualenv builds being broken due to Sphinx removing deprecated APIs.

### CODE REVIEW

**Comment by richvdh** [comment]
> Shpinx

schhhhpinx

File: docker/Dockerfile-dhvirtualenv [hunk 1]
```diff
@@ -47,8 +47,9 @@ RUN apt-get update -qq -o Acquire::Languages=none \
     && cd /dh-virtualenv \
     && env DEBIAN_FRONTEND=noninteractive mk-build-deps -ri -t "apt-get -y --no-install-recommends"
 
-# build it
-RUN cd /dh-virtualenv && dpkg-buildpackage -us -uc -b
+# Build it. Note that building the docs doesn't work due to differences in
```
Comment (richvdh) [inline docker/Dockerfile-dhvirtualenv:50]:
```suggestion
# Build it. Note that building the docs don't work due to differences in
```

**Review by richvdh — APPROVED** [review]
fine, but where is the updated sphinx coming from on bionic? https://packages.ubuntu.com/bionic/python3-sphinx says it hasn't changed since 2018.

**Comment by erikjohnston** [comment]
> fine, but where is the updated sphinx coming from on bionic? https://packages.ubuntu.com/bionic/python3-sphinx says it hasn't changed since 2018.

I can't now seem to repro it failing on bionic, but on sid the package was updated on Saturday: python3-sphinx from 4.1 to 4.2

**Pull request merged**